run

```diff
@@ -67,6 +67,28 @@ if [ ! -z "$OPENDKIM_DOMAINS" ] ; then
   service opendkim start
 fi
 service postfix start
+
+# Don't fiddle with existing config file (e.g. mounted from filesystem)
+if [ -e /etc/rsyslog.conf ]; then
+  echo "Skipping /etc/rsyslog.conf generating - file already exists"
+else
+  # Rsyslog base
+  cat <<'EOF' > /etc/rsyslog.conf
+$ModLoad imuxsock
+$WorkDirectory /var/spool/rsyslog
+*.*;auth,authpriv.none /dev/stdout
+EOF
+
+  if [ "${RSYSLOG_TIMESTAMP}" == 'no' ] ; then
+    echo '$template noTimestampFormat,"%syslogtag%%msg%\n"' >> /etc/rsyslog.conf
+    echo '$ActionFileDefaultTemplate noTimestampFormat' >> /etc/rsyslog.conf
+  fi
+
+  if [ "${RSYSLOG_LOG_TO_FILE}" == 'yes' ] ; then
+    echo 'mail.* -/var/log/mail.log' >> /etc/rsyslog.conf
```

Comment:
Should be same ”facilities” as stdout?

Reply:
Or document that it’s only mail?

Reply:
I will probably document that. I don't think there would be any other logs entries in syslog, as there are no other processes started in container than postfix, but it should be clearly stated.

Reply:
👍.  sounds good, push an update and i'll merge

Reply:
Added few words to readme